SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -467,6 +464,37 @@ void UGlobalStateManager::QueryGSM(const QueryDelegate& Callback)
 	Receiver->AddEntityQueryDelegate(RequestID, GSMQueryDelegate);
 }
 
+void UGlobalStateManager::QueryTranslation()
```

Comment:
This should likely be done in the Translator long term, but then the translator needs extra framework to send queries, while the GSM was already doing it (and ironically also had the code for processing the ops). To minimize the change in the RC, I decided to do this here. If anyone feels strongly that it should be in the Translator immediately, I'm happy to move it. Otherwise filing a ticket to move it in master after the RC is done.

Reply:
Is this the reason for making this into a new function, instead of just using the existing `QueryGSM` ?

Reply:
That's right. I figured it would be easier to move if it's already separate.